apim/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: apim
-version: 1.29.1
+version: 1.30.0
```

Comment:
Can we consider a 1.29.2 version instead please?

The rest of the PR is ok for me.

Reply:
Changed